dataaccess/cassandra/index.js

```diff
@@ -95,35 +103,56 @@ module.exports = function (settings) {
 		if (collectionname ==='n_flightSegment')
 			return [doc.originPort, doc.destPort, doc._id, JSON.stringify(doc)];
 		return [doc._id, JSON.stringify(doc)];
-
 	}
-	module.findOne = function(collectionname, key, callback /* (error, doc) */) {
-		var query = findByIdStmt[collectionname];
-		if (! query) {
-			callback ("FindById not supported on "+collectionname, null);
-			return;
+
+	module.findOne = async (collectionname, key) => {
+	  var query = findByIdStmt[collectionname];
+	  if (!query) {
+		throw ("FindById not supported on " + collectionname)
```

Comment:
should be an error

Reply:
@mafintosh 
Why `throw` is not working here? Do you want me to add `logger.error()` here or something else? I will check that every exception is handled or rethrown. Likely some are not yet. Still, I was thinking about dealing with this at routes level.
Please advise.